--- src/environments/environment.prod.ts	2021-04-23 11:57:19.000000000 +0200
+++ src/environments/environment.prod.ts	2021-04-23 12:17:50.000000000 +0200
@@ -1,3 +1,4 @@
 export const environment = {
-  production: true
+  production: true,
+  occBaseUrl: undefined
 };
--- src/environments/environment.ts	2021-04-23 11:57:19.000000000 +0200
+++ src/environments/environment.ts	2021-04-23 12:17:59.000000000 +0200
@@ -3,7 +3,8 @@
 // The list of file replacements can be found in `angular.json`.
 
 export const environment = {
-  production: false
+  production: false,
+  occBaseUrl: 'https://localhost:9002'
 };
 
 /*
