--- src/app/spartacus/features/smartedit/smart-edit-feature.module.ts	2021-07-16 09:53:45.000000000 +0200
+++ src/app/spartacus/features/smartedit/smart-edit-feature.module.ts	2021-07-16 09:52:35.000000000 +0200
@@ -1,6 +1,6 @@
 import { NgModule } from '@angular/core';
 import { CmsConfig, provideConfig } from "@spartacus/core";
-import { SmartEditRootModule, SMART_EDIT_FEATURE } from "@spartacus/smartedit/root";
+import { SmartEditConfig, SmartEditRootModule, SMART_EDIT_FEATURE } from "@spartacus/smartedit/root";
 
 @NgModule({
   declarations: [],
@@ -14,6 +14,12 @@
           import('@spartacus/smartedit').then((m) => m.SmartEditModule),
       },
     }
-  })]
+  }),
+  provideConfig(<SmartEditConfig>{
+    smartEdit: {
+      allowOrigin: 'localhost:9002, *.*.model-t.cc.commerce.ondemand.com:443',
+    },
+  })
+  ]
 })
 export class SmartEditFeatureModule { }
