--- src/app/spartacus/spartacus-configuration.module.ts	2021-09-11 07:58:00.000000000 +0200
+++ src/app/spartacus/spartacus-configuration.module.ts	2021-09-11 08:03:48.000000000 +0200
@@ -2,6 +2,7 @@
 import { translationChunksConfig, translations } from "@spartacus/assets";
 import { FeaturesConfig, I18nConfig, OccConfig, provideConfig, SiteContextConfig } from "@spartacus/core";
 import { defaultCmsContentProviders, layoutConfig, mediaConfig } from "@spartacus/storefront";
+import { environment } from 'src/environments/environment';
 
 @NgModule({
   declarations: [],
@@ -10,10 +11,14 @@
   providers: [provideConfig(layoutConfig), provideConfig(mediaConfig), ...defaultCmsContentProviders, provideConfig(<OccConfig>{
     backend: {
       occ: {
+        baseUrl: environment.occBaseUrl
       }
     },
   }), provideConfig(<SiteContextConfig>{
     context: {
+      // TODO: adjust parameters and site
+      // urlParameters: ['baseSite', 'language', 'currency'],
+      // baseSite: ['electronics-spa'],
       currency: ['USD'],
       language: ['en'],
     },
